src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: $card-width;
+
+   &__img {
+    display: block;
+
+    max-width: 160px;
+    width: 100%;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+   }
+
+   &__title {
+    font-family: Roboto, sans-serif;
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin: 0;
```

Comment:
You can use `reset.sccs` file to apply `margin: 0;` for `<h2/>`

Read more about `reset.scss` and `normalize.sccs` files:

https://elad.medium.com/normalize-css-or-css-reset-9d75175c5d1e

https://www.geeksforgeeks.org/difference-between-resetting-and-normalizing-in-css/